app/controllers/carl_people_controller.rb

```diff
@@ -0,0 +1,47 @@
+class CarlPeopleController < ApplicationController
+  #----------------- SHOW A PERSON --------------------
+  def show
+    @person = Person.find(params[:id])
+    @tasks_count = @person.tasks.count
+    @tasks_wip = @person.tasks.where(date_complete: nil).count
+    @tasks_done = @tasks_count - @tasks_wip
+  rescue
```

Comment:
I had tried doing it as a begin/rescue block, but it seemed to keep trying to execute the code after that point. Is that something that only happens in development mode but in production would just send up the 404 page? I did try pushing to heroku just now with a begin/rescue around line 4, but it sent a 500 error instead.

...this is starting to seem like I need to dive a little further into the different HTTP error codes. And maybe also Better Errors to see if it already throws the HTTP error codes or has a setting I can change so it will. I'll add these to my homework for this weekend.